feat(llc): add `fileSize`, `mimeType` in Attachment model (#689)
GetStream/stream-chat-flutter: packages/stream_chat/CHANGELOG.md

- Added `Filter.contains` and `Filter.empty`
- Added support for `next`, `previous` value pagination in `client.search`
  , [read more.](https://getstream.io/chat/docs/other-rest/search/#pagination)

> - `Attachment` class now have a `fileSize` and `mimeType` property. Setting an `file` will also set the `file_size`
>   , `mime_type` key on `extraData`, so `user.fileSize`, `user.mimetype` and `user.extraData['file_size']`
>   , `user.extraData['mime_type]` is same respectively.

Reviewer: ```suggestion
- `Attachment` class now has a `fileSize` and `mimeType` property. Setting a `file` will also set the `file_size`
  , `mime_type` key on `extraData`, so `attachment.fileSize`, `attachment.mimetype` and `attachment.extraData['file_size']`
  , `attachment.extraData['mime_type]` is same respectively.
```
